sw/host/README.md

```diff
@@ -1,3 +1,8 @@
 # Host Software
 
-- [Generated documentation for `opentitanlib`](https://opentitan.org/gen/rustdoc/opentitanlib)
+- [OpenTitanLib](./opentitanlib/README.md)
+- [OpenTitanTool](./opentitantool/README.md)
+- [OpenTitanSession](./opentitansession/README.md)
+- [OpenTitan Certificate Generator](./ot_certs/README.md)
+- [Hardware Security Module (HSM) tool](./hsmtool/README.md)
+- [TPM2 Test Server](./tpm2_test_server/README.md)
```

Comment:
WDYT about re-ordering this list to match the order in the SUMMARY.md navbar?

Reply:
That's a good idea. Changed the order in the navbar